src/rules/util.js

```diff
@@ -77,25 +77,9 @@ export const argument = node =>
 export const argument2 = node =>
   node.parent.parent.parent.arguments && node.parent.parent.parent.arguments[0];
 
-const describeAliases = new Set([
-  'describe',
-  'describe.only',
-  'describe.skip',
-  'fdescribe',
-  'xdescribe',
-]);
+const describeAliases = new Set(['describe', 'fdescribe', 'xdescribe']);
 
-export const testCaseNames = new Set([
-  'fit',
-  'it',
-  'it.only',
-  'it.skip',
-  'test',
-  'test.only',
-  'test.skip',
-  'xit',
-  'xtest',
-]);
+export const testCaseNames = new Set(['fit', 'it', 'test', 'xit', 'xtest']);
```

Comment:
That's a very good point about the export - I'll check it right now.

Definitely still catches `test.only`, b/c otherwise `no-focused-tests` would fail :)

All of the places that actually do something specific for things like `only` & `skip` actually implement their own checks; which makes sense b/c there's no way to explicitly get them out of the `Set`.

That's why it all still works, b/c the sets & guards are meant to serve as a way to know if you're in the "jest domain" (and so if you should bother checking anything at all), rather than for explicit "is it this specific jest method".